services/meta-extraction/src/index.js

```diff
@@ -0,0 +1,36 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import { extractAbstract } from '@openreview/meta-extraction';
+
+// Create an Express application
+const app = express();
+
+// Middleware setup
+app.use(bodyParser.json());
+
+// Define routes
+app.get('/metadata', async (req, res) => {
+  if (!req.query.url) {
+    return res.status(400).json({
+      name: 'ValidationError',
+      message: 'URL is required',
+      status: 400
+    });
+  }
+  const { abstract, pdf, error } = await extractAbstract(req.query.url, !!req.query.skipTidy);
```

Comment:
i think skipTidy won't be passed as query param
it should always be true

Reply:
sorry i mean it should always be false

Reply:
Oh OK! I can make the change

Reply:
DONE!